client/src/FIleToolbar.js

```diff
@@ -0,0 +1,321 @@
+
+import {
+  useData,
+} from 'flight-webapp-components';
+
+import { Context as FileManagerContext } from './FileManagerContext';
+import styles from './FileManager.module.css';
+
+function FileToolbar() {
+  return (
+    <div className={classNames("btn-toolbar", "flex-grow-1", styles.FileToolBar)}>
+      <NavButtons />
+      <NewFileFolderButtons />
+      <SelectionButtons />
+      <OperationsButtons />
+      <CopyPasteButtons />
+      <UpDownloadButtons />
+      <TarballButtons />
+      <BookmarkButtons />
+    </div>
+  );
+}
+
+function BookmarkButtons() {
+  const { navigate } = useContext(FileManagerContext);
+
+  const data = useData()
+  var bookmarks = data("bookmarks")
+  bookmarks = (bookmarks == null || !Array.isArray(bookmarks)) ? [] : bookmarks;
+
+  const items = bookmarks.map(item =>
+    <DropdownItem
+      className={styles.DropdownItem}
+      onClick={() => {navigate({path: item.path})}}
+      key={item.path}
+    >
+      <span className={classNames(`fa fa-${item.fa_icon}`, styles.fa)} />
+      <span className="title">{item.text}</span>
+    </DropdownItem>
+  );
+
+  const disabled = bookmarks.length === 0;
+  const title = (disabled) ? "No bookmarked directories" : "Go to a bookmarked directory"
+
+  return (
+    <div className={classNames('d-flex', 'flex-column', 'justify-content-center', 'align-items-center', styles.ButtonGroupWrapper)}>
+      <p className={styles.ButtonGroupTitle}>Bookmark</p>
+      <UncontrolledDropdown
+        className={classNames("btn-group", styles.ButtonGroup)}
+      >
+        <DropdownToggle
+          color="light"
+          size="sm" 
+          title={title}
+          disabled={disabled}
+          caret
+          split
+        >
+          <i className="fa fa-bookmark" />
+        </DropdownToggle>
+        <DropdownMenu>
+          {
+            items
+          }
+        </DropdownMenu>
+      </UncontrolledDropdown>
+    </div>
+  );
+}
+
+function NavButtons() {
+  const { goToParentDir, isRootDir } = useContext(FileManagerContext);
+  return (
+    <div className={classNames('d-flex', 'flex-column', 'justify-content-center', 'align-items-center', styles.ButtonGroupWrapper)}>
+      <ButtonGroup className={styles.ButtonGroup}>
+        <Button
+          color="light"
+          disabled={isRootDir}
+          onClick={goToParentDir}
+          size="lg"
+          title="Go to parent directory"
+        >
+          <i className="fa fa-level-up" />
+        </Button>
+      </ButtonGroup>
+    </div>
+  );
+}
+
+function NewFileFolderButtons() {
+  const { promptNewDir, promptNewFile } = useContext(FileManagerContext);
+  return (
+    <div className={classNames('d-flex', 'flex-column', 'justify-content-center', 'align-items-center', styles.ButtonGroupWrapper)}>
+      <p className={styles.ButtonGroupTitle}>New</p>
+      <ButtonGroup className={styles.ButtonGroup}>
+        <Button
+          color="light"
+          onClick={promptNewFile}
```

Comment:
A bigger change that isn't super important so don't worry if there's not enough time to change it...

The styling of the modals for naming new files and directories doesn't match the modals from other apps. Here's an example from Flight Desktop...
 
![image](https://github.com/openflighthpc/flight-file-manager/assets/102584263/235e1481-0f0b-4554-a89d-a929e3b71d4e)

If possible it would be good if the modal styling could be consistent

Reply:
Good spot I'm afraid it might not be changed until next time.